src/Microsoft.DotNet.VersionTools/project.json

```diff
@@ -1,5 +1,8 @@
-{
+﻿{
   "dependencies": {
+    "Microsoft.Build.Framework": "0.1.0-preview-00022",
+    "Microsoft.Build.Tasks.Core": "0.1.0-preview-00022",
+    "Microsoft.Build.Utilities.Core": "0.1.0-preview-00022",
```

Comment:
+1 on this.  IMO - it is good to split the actual implementation out from MSBuild, so we can write tools outside of MSBuild that do these operations.

For example, if we wanted to write a tool that consumed one of these "produces.json" files and made a chart/graph/website out of it.  If we had an object model for the "produces.json" structure split away from MSBuild, this would be easy.  And then have MSBuild Tasks that use this object model as necessary.